src/manifests_rbd.py

```diff
@@ -80,9 +80,7 @@ def __call__(self) -> Optional[AnyResource]:
             "csi.storage.k8s.io/node-stage-secret-namespace": ns,
             "csi.storage.k8s.io/provisioner-secret-name": StorageSecret.SECRET_NAME,
             "csi.storage.k8s.io/provisioner-secret-namespace": ns,
-            "imageFeatures": "layering",
             "pool": f"{self.fs_type}-pool",
-            "thickProvision": "false",
```

Comment:
this has been deprecated since `v3.6.0`.  It has no business being there.

Reply:
this may result in situations where the charm fails with an error  about not being able to patch the storage class.  So we've added this handy action to delete the storage class:

```bash
juju run ceph-csi/leader delete-storage-class name=ceph-xfs
juju run ceph-csi/leader sync-resources
```

Reply:
if it can't delete the storage class, we're in a real situation here where either the config needs to change to match the installed storage class or the cluster admin needs to check if there are PVCs using that storage class which is incorrectly provisioned